ptz-explainer.md

```diff
@@ -0,0 +1,166 @@
+# Explainer: Pan/Tilt support for Camera
+
+## Authors:
+
+- fbeaufort@google.com
+
+## Participate
+
+- https://github.com/w3c/mediacapture-image/issues
+
+## Introduction
+
+[Some cameras] have the ability to pan and tilt which is especially useful in
+video conferencing.
+
+The "Pan/Tilt support for Camera" feature will allow web developers to use this
+ability in a standards compliant manner using media track constraints in
+[MediaDevices.getUserMedia()] and [MediaStreamTrack.applyConstraints()].
+
+## Goals
+
+The [MediaStream Image Capture API](https://w3c.github.io/mediacapture-image/)
+has been [implemented in Chrome](https://caniuse.com/#search=imageCapture) since
+2017. We would like to add the missing pan and tilt camera motion features to
+the existing property set of capabilities, constraints and settings.
+
+This will allow video conferencing and real-time meetings web apps to control
+camera pan and tilt motions, aka PTZ (Pan/Tilt/Zoom). It could be used for
+steering the camera to face a speaker for instance.
+
+Without it, users have to install native apps today to control camera PTZ.
+Presently, zoom is supported as a media track constraint behind the camera
+permission. The plan is to group pan, tilt and zoom together behind a PTZ
+separate permission.
+
+## Control camera pan/tilt
+
+We would like to add `pan` and `tilt` to the existing property set of media
+track capabilities, constraints and settings. These new constraints apply to the
+live video feed. Those will also be used in `getUserMedia()` to express whether a
+website wants to control camera PTZ functionality. In other words, it will be
+used to request the PTZ permission, a separate camera permission, in a single
+call. If the selected/connected camera does not support PTZ though, the PTZ
+permission won’t be requested and may fallback to the camera permission.
+
+The [new "true" semantics] for `pan`, `tilt`, and `zoom` makes it possible to
+acquire a PTZ camera in `getUserMedia()` without altering the current pan, tilt
+and zoom values.
+
+Applying PTZ constraints requires the PTZ permission to be granted as described
+in the "Interaction with the Permissions API" section below. If not granted,
+user will be prompted. Otherwise, the pan and tilt values passed in
+`applyConstraints()` will be used to move the camera.
+
+The example below shows how camera pan and tilt could be requested and presented
+to the user.
+
+```js
+// User is prompted to grant camera PTZ access only if the camera supports PTZ.
+// If the camera does not support PTZ, it falls back to a "regular" camera prompt.
```

Comment:
But what if the user doesn't want anyone to remotely control their camera but are fine with accepting a video stream, but just not PTZ features? I think it would be nice to tread this as a "Yes, I accept to share my camera, but disallow people controlling it remotely" - the explainer could discuss this option and it would be great if the API would allow that as the same call. Just fallback to regular camera, and not regular camera prompt.

Like

"site would like to use your camera and additionally control it remotely [block] [allow] [allow just camera]"

Or something like

```
Site would like to use your camera.

[x] Also, allow site to remotely control your camera

[block] [allow]
```